eclipse-collections-code-generator/src/main/resources/api/list/primitiveList.stg

```diff
@@ -125,6 +131,29 @@ public interface <name>List extends Reversible<name>Iterable
     {
         throw new UnsupportedOperationException("Default method to prevent breaking backwards compatibility");
     }
+<if(primitive.specializedStream)>
+
+    /**
+     * @since 10.0
+     */
+    Spliterator.Of<name> spliterator();
+
+    /**
+     * @since 10.0
+     */
+    default <name>Stream primitveStream()
+    {
+        return StreamSupport.<type>Stream(this.spliterator(), false);
+    }
+
+    /**
+     * @since 10.0
+     */
+    default <name>Stream primitveParallelStream()
+    {
+        return StreamSupport.<type>Stream(this.spliterator(), true);
```

Comment:
If primitiaveParallelStream returned a non-parallel stream, would any tests break? Seems hard to test.

Reply:
it is not as hard because AbstractPipeline has  isParallel function. It just makes the tests longer. I can work to do this. will ad the test in primitiveListsTest.stg